build.sh

```diff
@@ -93,8 +93,11 @@ cd ../..
 
 echo ""
 echo "Copying extra plugins..."
-cp -r plugins/* ckeditor/plugins/
-
+if [ "$(uname)" == "Darwin" ]; then
+	cp -R plugins/ ckeditor/plugins/
```

Comment:
Well, someone has to check it on Linux and Win (cc @mlewand) to confirm. I did not want to mess up with the old command that used to work there.